doc/en/user/source/security/passwd.rst

```diff
@@ -97,8 +97,14 @@ By default, the master password is generated and stored in a file named ``securi
 
 .. warning:: The file ``security/masterpw.info`` is a security risk. The administrator should read this file and verify the master password by logging on GeoServer as the ``root`` user. On success, this file should be removed.
 
+.. warning:: First thing to do as an Administrator of the System, would be to dump the Master Password generated by GeoServer, store it somewhere not accessible by anyone, and delete any ``security/masterpw.info`` or whatever file you used to dump the password in clear.
+
```

Comment:
This makes line 98, above, redundant. That line should be removed. 

This note should link to instructions for how to dump the master password.

I would also suggest some wording improvements:

> The first thing an Administrator of the System should do is dump the Master Password generated by GeoServer, store it somewhere not accessible by anyone, and delete ``security/masterpw.info`` or whatever file you dumped the password to.